web_app/database/crud.py

```diff
@@ -0,0 +1,58 @@
+from typing import Any
+
+from sqlalchemy.orm import Session
+
+from utils.values import NotificationValidationValues
+
+
+def _exists_in_db(
+    db: Session = None,
+    model: Any = None,
```

Comment:
Please, be more specific with annotation. 
It will help you to identify errors in the futures.
For example:
```
`model` - most probably `Base` from sqlalchemy
`attr` will be as a string type
```